pom.xml

```diff
@@ -64,13 +64,13 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.3.5</version>
+            <version>1.3.14</version>
```

Comment:
Bumpe til 1.4.14 like gjerne? (og tilsvarende for logback-core)

Reply:
Tør ikkje.

Reply:
Good call 😛  logback 1.4 krever Java 11.